README.md

```diff
@@ -439,18 +445,35 @@ class AppMonitor < Karafka::Monitor
 
   private
 
-  def notice_consume(_caller_class, options)
-    record_count metric_key(options[:topic], __method__)
+  # Log that messages for a given topic were consumed
+  # @param topic [String] topic name
+  def consume(topic)
+    record_count metric_key(topic, __method__)
   end
 
-  def notice_perform_async(caller_class, _options)
-    record_count metric_key(caller_class, __method__)
+  # Log that message for topic were scheduled to be performed async
+  # @param topic [String] topic name
+  def perform_async(topic)
```

Comment:
Since now `perform_async`, `consume` and `perform` methods have the same body, maybe it is better to use aliases?
Or there the documentation matters? Anyway we can add documentation to aliases.